packages/nc-gui-v2/components/monaco/Editor.vue

```diff
@@ -73,8 +93,15 @@ onMounted(() => {
 watch(
   () => modelValue,
   (v) => {
-    if (editor && v && !deepCompare(v, JSON.parse(editor?.getValue() as string))) {
-      editor.setValue(JSON.stringify(v, null, 2))
+    if (editor && v) {
+      const value = editor?.getValue()
+      if (typeof v === 'object') {
```

Comment:
Do JSON parse only if `lang` is `json`

Reply:
Since it's an object, we need to parse it like that in order to compare nested values.

Reply:
I hope only in the case of JSON it will be an object.